qa_python_utils/aws/athena.py

```diff
@@ -319,6 +318,37 @@ def __create_athena_table(self, database, table_name, schema, location, serde, p
     def msck_repair_table(self, database, table_name):
         self.execute_query_and_wait_for_results("""MSCK REPAIR TABLE {}.{}""".format(database, table_name))
 
+    @logger
+    def upsert_partitions(self, bucket_folder_path, database, table, partition_name_list, partition_value_list):
```

Comment:
do you think it could be safer/clearer to have a list of dicts, instead of two lists? that way we guarantee that the name will be linked to the value and will have only one param.
for example:
```suggestion
    def upsert_partitions(self, bucket_folder_path, database, table, partitions_dict):
```

```
athena_client.upsert_partitions(
    bucket_folder_path=...,
    database=...,
    table=....,
    partitions_list_dicts = [
        { 'name0': 'value0'},
        { 'name1': 'value1'},
                ....
    ]
)
```

Reply:
done in commit: 77e67505ba8fe9a521ffb0f8b5cb2996d64a39b5

Reply:
updated in asterisk class: https://github.com/quintoandar/bi-etl-ejuice/commit/8658deed1689f75b2fb0412822e41c61ecbe1138